drivers/serial/uart_nrf5.c

```diff
@@ -263,19 +108,17 @@ static int uart_nrf5_init(struct device *dev)
  * @return 0 if a character arrived, -1 if the input buffer if empty.
  */
 
-static int uart_nrf5_poll_in(struct device *dev, unsigned char *c)
+static inline int uart_nrf5_poll_in(struct device *dev, unsigned char *c)
```

Comment:
These "inline" keywords were supposed to disappear completely. Three of them still appear in this commit and are removed in the second one.